src/app/CreateRecording/CreateRecording.tsx

```diff
@@ -0,0 +1,262 @@
+              />
+            </FormGroup>
+            <FormGroup
+              label="Duration"
+              isRequired
+              fieldId="recording-duration"
+            >
+              <Checkbox
+                label="Continuous"
+                isChecked={continuous}
+                onChange={handleContinuousChange}
+                aria-label="Continuous checkbox"
+                id="recording-continuous"
+                name="recording-continuous"
+              />
+              <Split gutter="md">
+                <SplitItem isFilled>
+                  <TextInput
+                    value={duration}
+                    isRequired
+                    type="number"
+                    id="recording-duration"
+                    aria-describedby="recording-duration-helper"
+                    onChange={handleDurationChange}
+                    isDisabled={continuous}
+                    min="0"
+                  />
+                </SplitItem>
+                <SplitItem>
+                  <FormSelect
+                    value={durationUnit}
+                    onChange={handleDurationUnitChange}
+                    aria-label="Duration Units Input"
+                    isDisabled={continuous}
+                  >
+                    <FormSelectOption key="1" value="1" label="Seconds" />
+                    <FormSelectOption key="2" value={60} label="Minutes" />
+                    <FormSelectOption key="3" value={60*60}label="Hours" />
+                  </FormSelect>
+                </SplitItem>
+              </Split>
+            </FormGroup>
+            <FormGroup
+              label="Events"
+              isRequired
+              fieldId="recording-events"
+              isValid={eventsValid}
+              helperText="Select a template from the dropdown, or enter a template name or event specifier string in the text area."
+            >
+              <Split gutter="md">
+                <SplitItem>
+                  <FormSelect
+                    value={template}
+                    onChange={handleTemplateChange}
+                    aria-label="Event Template Input"
+                  >
+                    <FormSelectOption key="0" value="" label="Custom Event Definition" />
+                    <FormSelectOptionGroup key="1" label="Remote Templates">
+                      {
+                        templates.map((t: EventTemplate, idx: number) => (<FormSelectOption key={idx+2} value={t.name} label={t.name} />))
+                      }
+                    </FormSelectOptionGroup>
+                  </FormSelect>
+                </SplitItem>
+                <SplitItem>
+                  <Tooltip
+                    position={TooltipPosition.auto}
+                    content={
+                      <Text component={TextVariants.p}>
+                        Templates are selected with the syntax <i>template=Foo</i>. Event names and options are specified with the syntax
```

Comment:
For whatever reason, when I view the tooltip I see
"with the syntax*namespace.EventName...*"

I guess you need an explicit space here?

Reply:
I reformatted the tooltip. It should be easier to read now and that odd lack of whitespace probably won't happen - although I'm not sure why the .tsx template would have interpreted the text that way anyway.

Reply:
Nice!